website/tables/azure/azure_servicebus_namespace_topic_rule_access_keys.md

```diff
@@ -0,0 +1,27 @@
+# Table: azure_servicebus_namespace_topic_rule_access_keys
+
+https://learn.microsoft.com/en-us/rest/api/servicebus/stable/topics%20%E2%80%93%20authorization%20rules/list-keys?tabs=HTTP#accesskeys
+
+The composite primary key for this table is (**rule_id**, **key_name**).
+
+## Relations
+
+This table depends on [azure_servicebus_namespace_topic_authorization_rules](azure_servicebus_namespace_topic_authorization_rules).
+
+## Columns
+
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|String|
+|_cq_sync_time|Timestamp|
+|_cq_id|UUID|
+|_cq_parent_id|UUID|
+|subscription_id|String|
+|rule_id (PK)|String|
+|alias_primary_connection_string|String|
+|alias_secondary_connection_string|String|
+|key_name (PK)|String|
```

Comment:
Key names are unique based on my tests (create a key with the same name using the CLI overrides the previous one).

The rule ID has the subscription, namespace and topic built in, along with key name should be a good PK